docs/usage.md

```diff
@@ -1,6 +1,6 @@
 # Usage
 
-TIP: **This is the documentation for the NEW Python handler.**  
```

Comment:
The trailing spaces are here on purpose. Or is this way of adding a line break deprecated?